spec/controllers/conversations_controller_spec.rb

```diff
@@ -102,6 +102,14 @@
       get :index, conversation_id: @conversations.first.id
       expect(assigns[:conversation]).to be_nil
     end
+
+    it "retrieves a conversation message with out markdown content " do
+      get :index
+      @conversation = @conversations.first
+      expect(response).to be_success
+      expect(response.body).to match(%r/cool stuff/)
+      expect(response.body).not_to match(%r/<strong>cool stuff<\/strong>/)
```

Comment:
See my comment. This should be `%r{<strong>cool stuff</strong>}`. (Note the unescaped slash!)